tests/examples/test_example_bots_training_data.py

```diff
@@ -0,0 +1,42 @@
+from pathlib import Path
+from typing import Text, List
+
+import pytest
+
+from rasa.importers.importer import TrainingDataImporter
+from rasa.importers.utils import story_graph_from_paths
+
+EXAMPLES_FOLDER_ROOT = "examples"
+EXAMPLES_FOLDER_SUFFIX = "data"
+CONFIG_FILE = "config.yml"
+DOMAIN_FILE = "domain.yml"
+
+
+def get_training_data_folder(bot_name: Text) -> List[Text]:
+    return [str(Path(EXAMPLES_FOLDER_ROOT) / bot_name / EXAMPLES_FOLDER_SUFFIX)]
+
+
+def get_config_file(bot_name: Text) -> Text:
+    return str(Path(EXAMPLES_FOLDER_ROOT) / bot_name / CONFIG_FILE)
+
+
+def get_domain_file(bot_name: Text) -> Text:
+    return str(Path(EXAMPLES_FOLDER_ROOT) / bot_name / DOMAIN_FILE)
+
+
+@pytest.mark.parametrize(
+    "bot_name", ["concertbot", "knowledgebasebot", "moodbot", "reminderbot", "rules"]
```

Comment:
OPT: as much as I like your DRY approach, this breaks IDEs "find where this is used" feature as well as manual searches - I think in terms of keeping things simple there is no harm in just listing the paths here

Reply:
Do you mean listing explicitly:
- `examples/concertbot`
- `examples/knowledgebasebot`

etc?

Reply:
Not sure how to not break "find usage" anyway 😕

Reply:
yes, I'd just list the files, e.g.
```
@pytest.mark.parametrize(
 "domain, config, ...", [
  ("examples/concertbot/domain.yml", "examples/concertbot/config.yml"),
  ("examples/knowledgebasebot/domain.yml","examples/knowledgebasebot/config.yml"),
  ...
```